usr/share/lib/ipa/tests/SLES/infra/test_sles_switch_smt.py

```diff
@@ -0,0 +1,27 @@
+import shlex
+
+
+def test_sles_switch_smt(GetSMTServerName,
+                         GetSMTServers,
+                         GetReleaseValue,
+                         host,
+                         request):
```

Comment:
Add doc string to indicate that this i snot actually a test, but rather a helper function cast as a test to make it easily includable in other tests.